Makefile

```diff
@@ -46,15 +46,15 @@ RELEASE_OUT ?=
 TELEPORT_TAG = 3.2.16
 # TELEPORT_REPOTAG adapts TELEPORT_TAG to the teleport tagging scheme
 TELEPORT_REPOTAG := v$(TELEPORT_TAG)
-PLANET_TAG := 7.1.12-$(K8S_VER_SUFFIX)
+PLANET_TAG := 7.1.12-$(K8S_VER_SUFFIX)-4-g829ffe85
 PLANET_BRANCH := $(PLANET_TAG)
 K8S_APP_TAG := $(GRAVITY_TAG)
 TELEKUBE_APP_TAG := $(GRAVITY_TAG)
 WORMHOLE_APP_TAG := $(GRAVITY_TAG)
 INGRESS_APP_TAG ?= 0.0.1
 STORAGE_APP_TAG ?= 0.0.3
 LOGGING_APP_TAG ?= 6.0.7
-MONITORING_APP_TAG ?= 7.0.2
+MONITORING_APP_TAG ?= 7.1.0-1-g918973e0
```

Comment:
this is not going to work as expected - I'd suggest using the name of the branch [here](https://github.com/gravitational/gravity/blob/bernard/master/ping/build.assets/Makefile#L30)